R/collectors.R

```diff
@@ -200,6 +200,7 @@ guess_parser <- function(x, locale = default_locale()) {
 #'
 #' @param levels Character vector providing set of allowed levels.
 #' @param ordered Is it an ordered factor?
+#' @param include_na Should `NA` be included as a factor level?
```

Comment:
I think this should be, "If `NA` are present, include as an explicit factor to level?".  And I think it should default to `TRUE`